src/models/attachment.ts

```diff
@@ -26,3 +26,24 @@ export const internalAttachmentCreateBodyParser = z.object({
   ownerAddress: z.string(),
 })
 export type InternalAttachmentCreateBody = z.infer<typeof internalAttachmentCreateBodyParser>
+
+export const uuidRegex = /^[0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i
+
+/**
+ * UUID format identifier
+ * @example 282439cd-ebc8-420e-bfbb-0de8fb3149ed
+ * @pattern ^[0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$
+ */
+export type AttachmentId = string
+
+/**
+ * IPFS Content Identifier (CID) hash
+ * @example QmX5g1GwdB87mDoBTpTgfuWD2VKk8SpMj5WMFFGhhFacHN
+ * @pattern ^[a-zA-Z0-9]{1,44}$
+ */
+export type AttachmentHash = string
+
+/**
+ * Either an attachment UUID or an IPFS hash
+ */
```

Comment:
I'd like to avoid refering to or assuming IPFS here. Can we therefore:

1. refer instead of `IPFS Hash` to a `file integrity hash`
2. Change the regex to something more forgiven that still doesn't overlap with a UUID, e,g, `^[a-zA-Z0-9]+$`